src/views/catalog/CreateVMHorizontalNav/components/CreateFromInstanceTypes/utils/constants.ts

```diff
@@ -0,0 +1,17 @@
+export enum INSTANCE_TYPES_SECTIONS {
+  SELECT_VOLUME = 1,
+  SELECT_INSTANCE_TYPE = 2,
+  VM_DETAILS = 3,
+}
+
+export const VirtualMachineClusterPreferenceGroupVersionKind = {
```

Comment:
Shouldn't this and the one below go in kubevirt-api? Are these just temporary until you get them added there?

Reply:
I added the proper models to kubevirt-api, and changed it